README.md

```diff
@@ -78,18 +78,18 @@ The FortiGate configurations deployed using `cloud-init` can be found under `Con
 
 The following is a guide to some common errors you may encounter.
 
-> Error: [ERROR] terraform-provider-alicloud/alicloud/resource_alicloud_log_project.go:61:
-> Resource alicloud_log_project CreateProject Failed!!! [SDK aliyun-log-go-sdk ERROR]
+> **Message**: Error: [ERROR] terraform-provider-alicloud/alicloud/resource_alicloud_log_project.go:61:
+> **Message**: Resource alicloud_log_project CreateProject Failed!!! [SDK aliyun-log-go-sdk ERROR]
```

Comment:
I don't **Message**: should be on this line?

Reply:
You are correct. Updated.